src/enigma2/Channels.cpp

```diff
@@ -281,7 +281,10 @@ int Channels::LoadChannelsExtraData(const std::shared_ptr<enigma2::data::Channel
 
     //We can use the JSON API so let's supplement the data with extra information
 
-    const std::string jsonURL = StringUtils::Format("%sapi/getservices?provider=1&picon=1&sRef=%s", Settings::GetInstance().GetConnectionURL().c_str(), WebUtils::URLEncodeInline(channelGroup->GetServiceReference()).c_str());
+    const std::string jsonURL = StringUtils::Format("%sapi/getservices?provider=1&picon=%d&sRef=%s",
```

Comment:
This one is changing the Picon parameter with toggle not Provider:
Off /api/getservices?provider=1&picon=0 
On /api/getservices?provider=1&picon=1

Reply:
Nice catch, thanks!